app/_src/gateway/production/sizing-guidelines.md

```diff
@@ -169,6 +170,17 @@ the better choice, as it provides near-unlimited throughput capacity. In this
 scenario, adding more cache memory would not increase maximum throughput by
 much.
 
+{% if_version gte:3.5.x %}
+In {{site.base_gateway}} setups with high cardinality of configured Routes, Services,
+Plugins, Consumers etc, the cluster may benefit from "Dedicated Config Processing". The
+feature presents an innovative approach to enhance system performance by employing a dedicated
+worker process. By offloading specific tasks, it promises reduced latencies and offers a more
+efficient operational experience, when operating in `hybrid mode``.
+Users should remain aware of the slight increase in memory usage but can benefit significantly
+from the accelerated processing times.
```

Comment:
```suggestion
When {{site.base_gateway}} is operating in `hybrid` mode with a large number of configuration 
entities (routes, services etc.), it can benefit from the "Dedicated Configuration Processing" option.  
When enabled, certain CPU  intensive steps of the data plane reconfiguration operation are offloaded 
to a dedicated worker process.  This reduces proxy latency during reconfigurations at the cost of a 
slight increase in memory usage.  The benefits of this mechanism are most apparent with configurations
consisting of more than a thousand configuration entities.
```